tools/api-docs/api-classes.ts

```diff
@@ -17,7 +17,7 @@
 
 import FS from 'node:fs/promises';
 import FSLib from '../libs/fs.js';
-import TSLib from '../libs/ts.js';
+import TSLib from '../libs/TS.js';
```

Comment:
`Error: Cannot find module '../libs/TS.js'`. This and the one in api-options should be changed from .js to no extension (`import TSLib from '../libs/TS';`).